macrostate.gap

```diff
@@ -26,9 +26,15 @@ include "Algebras/Probing/alg_probing_macrostate.gap" //an algebra for integrati
 
 algebra alg_count auto count ;
 algebra alg_enum auto enum ;
+algebra alg_tikz auto tikz;
+
 
 include "Grammars/gra_macrostate.gap"
 
+/*
+example inputs: CACUGUAAAGCUAACUUAGCAUUAACCUUUUAAGUUAAAGAUUAAGAGAACCAACACCUCUUUACAGUGACCA
+*/
+
```

Comment:
should this be in the same commit? not that it matters greatly.

Reply:
oh, I was not aware someone would read this :-) Thanks! This is special syntax to add a default input example when loaded on bellmanscafe.jlab.bio